lockable/lockable.py

```diff
@@ -228,7 +179,6 @@ def lock(self, requirements: (str or dict), timeout_s: int = 1000) -> Allocation
         :return: Allocation context
         """
         assert isinstance(self._resource_list, list), 'resources list is not loaded'
```

Comment:
Triggers http call.

Reply:
Also `self.logger.debug("Resource list: %s", json.dumps(self._resource_list))` triggers.

Reply:
fixed in 6f519e9